specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -502,19 +362,47 @@ The timespan of the query. It is a string with the following format
     Metrics
```

Comment:
Should MetricRequestPayload be called MetricsDimensionFilter?

Reply:
We have dimension filter as part of the metric request payload. For now the payload only contains filter but in future payload may contains other fields as well. Therefore keeping it MetricRequestPayload